### DISCUSSION TITLE
Not able to run in windows machine

### DISCUSSION THREAD

**Discussion by Piyush-Use-Personal** [chatwoot/chatwoot-mobile-app#366] [Q&A]
Hey, I came across this project and It seems a very clean and a good idea. I wanted to take a pull and run it on my machine to see everything in action but When I try to do it in macOS it seems good after changing few kinds of stuff but on windows, it does not work at all. Any thoughts? 
**Changes made**
- Added .env
- Changed to Gradle file
- increase the Gradle size to 4g 

It throws memory out of the bound error

**Comment by LukasdeSouza** [comment]
same here
